package.json

```diff
@@ -200,8 +200,8 @@
     "sinon-chai": "^3.7.0",
     "socket.io": "^4.6.0",
     "socket.io-client": "^4.6.0",
-    "sqs-consumer": "5.7.0",
-    "sqs-consumer-v6": "npm:sqs-consumer@^6.2.0",
+    "sqs-consumer": "^7.4.0",
+    "sqs-consumer-v5": "npm:sqs-consumer@^5.7.0",
```

Comment:
Question: Why do we test against v7 and v5? 🤔

Reply:
I don't mind testing against v5 and v7 if there is a reason 👍

Reply:
v5 is using aws sdk v2 and sqs v7 is using aws sdk v3

Reply:
Okay thx 👌 

I am wondering if we should drop support for aws sdk v2.

> NOTE: The AWS SDK for JavaScript (v2) will be put into maintenance mode in 2023.